.eslintrc.js

```diff
@@ -14,21 +14,26 @@ module.exports = {
   },
   "extends": [
     "eslint:recommended",
+    "plugin:jsx-a11y/recommended",
     "plugin:mozilla/recommended",
     "plugin:react/recommended",
   ],
   "env": {
     "browser": true,
+    "es6": true,
     "node": true,
     "webextensions": true,
-    "es6": true,
   },
   "plugins": [
+    "jsx-a11y",
     "mozilla",
     "react",
     ...extraPlugins,
   ],
   "rules": {
+    "jsx-a11y/click-events-have-key-events": "warn",
+    "jsx-a11y/no-noninteractive-element-interactions": "warn",
```

Comment:
I had to override these to be warnings since they were throwing CI breaking errors... Plus, I'm lazy and this was the "easy way out". #yolo

Reply:
Let's leave these as errors and wait until #98 lands.